mne/_freesurfer.py

```diff
@@ -422,6 +425,122 @@ def get_mni_fiducials(subject, subjects_dir=None, verbose=None):
     return fids
 
 
+@verbose
+def compute_mri_head_t(subject, subjects_dir, verbose=None):
+    """Get the head to surface RAS transform using the Freesurfer recon.
```

Comment:
```suggestion
    """Get the head to surface RAS transform using the Freesurfer reconstruction.
```

Reply:
It's too long unfortunately, that's the one-line summary

Reply:
Then please define the abbreviation below. I was unsure if you meant reconstruction or recon, as in a first exploratory observation.

Reply:
FYI, it's a holdover from the freesurfer command called `recon-all` I believe.  But I agree it would be nice to spell out "reconstruction" later in the docstring if there's a place where it makes sense to do so.